packages/components/segmented/src/segmented.vue

```diff
@@ -118,6 +118,7 @@ const updateSelect = () => {
   ) as HTMLElement
   if (!selectedItem || !selectedItemInput) {
     state.width = 0
+    state.height = 0
```

Comment:
You may need to rebase the code in the dev branch, it seems a bit problematic at the moment, carrying extraneous commit messages and file changes.

![image](https://github.com/user-attachments/assets/82a4847f-716c-47b9-a90e-25e4396905e5)